lsp-methods.el

```diff
@@ -1481,7 +1481,10 @@ interface RenameParams {
   (let ((edits (lsp--send-request (lsp--make-request
                                    "textDocument/rename"
                                    (lsp--make-document-rename-params newname)))))
-    (lsp--apply-workspace-edits edits)))
+    (if (sequencep edits)
+      (lsp--apply-workspace-edits edits)
```

Comment:
If I read it correctly, the `sequencep` case is not needed.

https://github.com/Microsoft/language-server-protocol/blob/master/versions/protocol-1-x.md#workspaceedit
https://github.com/Microsoft/language-server-protocol/blob/master/versions/protocol-2-x.md#workspaceedit
https://microsoft.github.io/language-server-protocol/specification

Thanks! It works perfectly with cquery.